script/dgl_dev.yml.template

```diff
@@ -25,5 +25,6 @@ dependencies:
     - torch==__TORCH_VERSION__
     - torchmetrics
     - tqdm
+    - boto3
```

Comment:
please add a comment in case we forget why we need this dependency.

Reply:
done